config/config_helpers.go

```diff
@@ -157,6 +165,33 @@ func pathRelativeToInclude(include *IncludeConfig, terragruntOptions *options.Te
 	return util.GetPathRelativeTo(currentPath, includePath)
 }
 
+// Return the relative path from the current Terragrunt configuration to the included Terragrunt configuration file
+func pathRelativeFromInclude(include *IncludeConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
+	if include == nil {
+		return ".", nil
+	}
+
+	relativePath, err := findInParentFolders(terragruntOptions)
+	if err != nil {
+		return "", errors.WithStackTrace(err)
+	}
+
+	return filepath.ToSlash(filepath.Dir(relativePath)), nil
+}
+
+// Return the AWS account id associated to the current set of credentials
+func getAWSAccountID(terragruntOptions *options.TerragruntOptions) (string, error) {
+	iamSession := iam.New(session.Must(session.NewSession()))
+
+	result, err := iamSession.ListUsers(&iam.ListUsersInput{MaxItems: aws.Int64(1)})
```

Comment:
Instead of calling `ListUsers`, I think [GetCallerIdentity](https://docs.aws.amazon.com/sdk-for-go/api/service/sts/#STS.GetCallerIdentity) is a better fit. It returns the account ID (so no need to parse the ARN) and it requires no extra permissions.